src/cockpit/polkit.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
+import locale
+import logging
+import os
+import pwd
+from typing import Dict, List, Sequence, Tuple
+
+from cockpit._vendor.ferny import InteractionResponder
+from cockpit._vendor.systemd_ctypes import Variant, bus
+
+# that path is valid on at least Debian, Fedora/RHEL, and Arch
+HELPER_PATH = '/usr/lib/polkit-1/polkit-agent-helper-1'
+
+AGENT_DBUS_PATH = '/agent'
```

Comment:
I'd have preferred a name with at least `polkit` or `authentication` in it somehow, but this is also fine :)